resources/dicts/conditions/pregnancy.json

```diff
@@ -1,6 +1,9 @@
 {
   "announcement": [
-    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits."
+    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits.",
+	  "m_c had been secretive for a while now and nobody could even begin to guess what about... until today, that is, when m_c announced {PRONOUN/m_c/poss} pregnancy!",
+          "The entire Clan could see the signs way before m_c did. Everyone urged {PRONOUN/m_c/object} to visit the medicine den to no avail. They just about gave up until, finally, m_c announced {PRONOUN/m_c/poss} pregnancy.",
```

Comment:
should be 'medicine cat den' not 'medicine den'

Reply:
Last time I made a pr referring to the location, it was requested I change it to medicine den